tools/src/test/java/org/apache/kafka/trogdor/common/WorkerUtilsTest.java

```diff
@@ -73,7 +73,7 @@ public void setUp() throws Exception {
     public void testCreateOneTopic() throws Throwable {
         Map<String, NewTopic> newTopics = Collections.singletonMap(TEST_TOPIC, NEW_TEST_TOPIC);
 
-        WorkerUtils.createTopics(log, adminClient, newTopics, true);
+        WorkerUtils.createTopics(log, adminClient, newTopics, true, 3, 2500);
```

Comment:
Reuse `DEFAULT_TOPIC_VERIFY_BACKOFF` that you defined?

Reply:
That's a good point.  I'll do that.

Reply:
I also added a single source of truth for the number of retries too.